pkg/network/kubemacpool.go

```diff
@@ -47,6 +53,19 @@ func renderKubeMacPool(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir str
 		return nil, nil
 	}
 
+	if conf.KubeMacPool.StartPoolRange == "" || conf.KubeMacPool.EndPoolRange == "" {
```

Comment:
please add an inline comment describing what happens here, mention how big the range is and the prefix for local unicast (IIUIC)

Reply:
done